docs/core/whats-new/dotnet-10/libraries.md

```diff
@@ -113,6 +113,73 @@ private static bool ValidateMLDsaSignature(ReadOnlySpan<byte> data, ReadOnlySpan
 
 The PQC algorithms are available on systems where the system cryptographic libraries are OpenSSL 3.5 (or newer) or Windows CNG with PQC support. Also, the new classes are all marked as [`[Experimental]`](../../../fundamentals/syslib-diagnostics/experimental-overview.md) under diagnostic `SYSLIB5006` until development is complete.
 
+#### ML-DSA enhancements
```

Comment:
The ML-DSA type is new for .NET 10, so it hasn't "gained enhancements". We should just describe the type as it exists as of this moment.